package.json

```diff
@@ -29,57 +35,99 @@
     "object-assign": "^4.0.1",
     "react-side-effect": "1.0.2",
     "shallowequal": "0.2.2",
-    "warning": "2.1.0"
+    "warning": "3.0.0"
   },
   "devDependencies": {
-    "babel": "5.8.34",
-    "babel-eslint": "5.0.0",
-    "babel-loader": "5.4.0",
-    "chai": "3.4.1",
-    "chalk": "1.1.1",
-    "del": "2.2.0",
-    "eslint": "2.4.0",
-    "eslint-config-nfl": "6.1.0",
-    "eslint-plugin-react": "4.2.2",
-    "estraverse": "^4.2.0",
-    "estraverse-fb": "^1.3.1",
-    "gulp": "3.9.0",
-    "gulp-cached": "1.1.0",
-    "gulp-debug": "2.1.2",
-    "gulp-eslint": "2.0.0",
-    "gulp-if": "2.0.0",
-    "gulp-notify": "2.2.0",
-    "gulp-util": "3.0.7",
-    "isparta-instrumenter-loader": "0.2.1",
-    "karma": "0.13.19",
+    "babel-cli": "6.16.0",
+    "babel-core": "6.16.0",
+    "babel-eslint": "7.0.0",
+    "babel-plugin-add-module-exports": "0.2.1",
+    "babel-plugin-transform-es2015-modules-commonjs": "6.16.0",
+    "babel-preset-latest": "6.16.0",
+    "babel-preset-react": "6.16.0",
+    "babel-preset-stage-0": "6.16.0",
+    "chai": "3.5.0",
+    "eslint": "3.6.1",
+    "eslint-config-nfl": "10.0.0",
+    "eslint-plugin-import": "1.16.0",
+    "eslint-plugin-jsx-a11y": "2.2.2",
+    "eslint-plugin-mocha": "4.5.1",
+    "eslint-plugin-react": "6.3.0",
+    "estraverse": "4.2.0",
+    "estraverse-fb": "1.3.1",
+    "isparta-loader": "2.0.0",
+    "karma": "1.3.0",
     "karma-chai": "0.1.0",
     "karma-chai-sinon": "0.1.5",
-    "karma-chrome-launcher": "0.2.2",
-    "karma-cli": "0.1.2",
-    "karma-coverage": "0.5.3",
+    "karma-chrome-launcher": "2.0.0",
+    "karma-coverage": "1.1.1",
     "karma-html-reporter": "0.2.7",
-    "karma-mocha": "0.2.1",
-    "karma-phantomjs-launcher": "0.2.3",
-    "karma-phantomjs-shim": "1.2.0",
-    "karma-sourcemap-loader": "0.3.6",
-    "karma-spec-reporter": "0.0.23",
+    "karma-mocha": "1.2.0",
+    "karma-phantomjs-launcher": "1.0.2",
+    "karma-phantomjs-shim": "1.4.0",
+    "karma-sourcemap-loader": "0.3.7",
+    "karma-spec-reporter": "0.0.26",
     "karma-tap-reporter": "0.0.6",
-    "karma-webpack": "1.7.0",
-    "mocha": "2.3.4",
-    "phantomjs": "1.9.19",
+    "karma-webpack": "1.8.0",
+    "mocha": "3.1.0",
+    "phantomjs-prebuilt": "2.1.12",
     "react": "15.x",
     "react-dom": "15.x",
-    "sinon": "1.17.2",
+    "sinon": "1.17.6",
     "sinon-chai": "2.8.0",
-    "webpack": "1.12.11"
+    "webpack": "1.13.2"
   },
   "scripts": {
-    "clean": "gulp clean",
-    "lint": "gulp eslint",
+    "clean": "rm -rf ./lib && rm -rf ./build && rm -rf ./es",
```

Comment:
Ok. Will add `rimraf` back in and also add `cross-env` so windows can handle `BABEL_ENV=commonjs`